### PR TITLE
Gcd python

### PR DESCRIPTION
**Pull request by sakshi300699**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
(Write your answer here.)

### Description

This is a python program to find the GCD of two numbers. I have used recursion to find the GCD.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue Addressed - #1902 
(Write your answer here.)

### CODE REVIEW

**Comment by sakshi300699** [comment]
Project Maintainers please review the PR.

File: Python/math/GCD_of_two_numbers.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Output:
+
+Enter 1st number: 48
+Enter 2nd number: 6
+The GCD is:  6
+
+Enter 1st number: 43
+Enter 2nd number: 54
+The GCD is:  1
+"""
```
Comment (HarshCasper) [inline Python/math/GCD_of_two_numbers.py]: Add a new line at the end of file

File: Python/math/GCD_of_two_numbers.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    if(x==0):
+        return y
+    else:
+        return gcd_of_nums(x,y%x)
+
+num1 = int(input("Enter 1st number: "))
+num2 = int(input("Enter 2nd number: "))
+
+print("The GCD is: ", gcd_of_nums(num1,num2))
```
Comment (HarshCasper) [inline Python/math/GCD_of_two_numbers.py]: Enclose this in a main function

File: Python/math/GCD_of_two_numbers.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#Program to find GCD of 2 Numbers in Python.
```
Comment (HarshCasper) [inline Python/math/GCD_of_two_numbers.py]: Add an Intro about the Algorithm that you have implemented in multi-line string

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by sakshi300699** [comment]
@HarshCasper sir, I have made the required changes, Kindly review.

**Comment by sakshi300699** [comment]
Project Maintainers, please review.

**Comment by sakshi300699** [comment]
@shraddhavp @iamrajiv please review the pr if any other change is needed

File: Python/math/GCD_of_two_numbers.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Enter 1st number: 48
+Enter 2nd number: 6
+The GCD is:  6
+
+Enter 1st number: 43
+Enter 2nd number: 54
+The GCD is:  1
+"""
+print("\n")
```
Comment (HarshCasper) [inline Python/math/GCD_of_two_numbers.py:41]: Not needed

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please squash your Commits and I would merge it.

**Comment by sakshi300699** [comment]
@HarshCasper Squashed the Commits, Kindly merge.

File: C-Plus-Plus/README.md [hunk 5]
```diff
@@ -184,6 +184,7 @@ _add list here_
 - [Zero One Knapsack](dp/Zero_One_Knapsack.cpp)
 - [Sprague Grundy Theorem on Strings](dp/Sprague_Grundy_Theorem.cpp)
 
+
```
Comment (iamrajiv) [inline C-Plus-Plus/README.md]: @sakshi300699 Do not edit this file.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@sakshi300699 Remove `e -i [SHA]` file.

**Comment by sakshi300699** [comment]
Okay I will make the necessary changes.

**Comment by sakshi300699** [comment]
@iamrajiv @HarshCasper I have made the necesssary changes. However there is some conflict with the PR for which I need to edit the C-Plus-Plus/README.md file. How should I proceed further?

**Comment by iamrajiv** [comment]
@sakshi300699 Yes resolve the conflicts by editing the `C-Plus-Plus/README.md` file.

**Comment by sakshi300699** [comment]
@iamrajiv @HarshCasper The conflict has been resolved. Please review.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**